src/formElementPlugins/longText/src/TemplateView.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react'
+import { Input } from 'semantic-ui-react'
+import { TemplateViewProps } from '../../types'
+
+// TODO type should be global
```

Comment:
Is this just copied from `shortText`? Not sure if that's required anymore (TODO's 👮‍♀️)

Reply:
Not touching "Template View" at the moment -- please just ignore the whole file.

Reply:
ok